README.md

```diff
@@ -122,17 +122,22 @@ _Context: global_
 
 Use this command to run tests on a package. Currently, there are two types of tests available.
 
+_Context: package_
+
```

Comment:
I would move this back to where it was. That way the two sections for the two types of tests follow right after the sentence, "Currently, there are two types of tests available."

Reply:
Fixed

I thought it might be confusing that the context refers only to system tests.

Reply:
Yeah, fair point. Maybe we should move all the _Context:_ lines below their respective command headings? Separate PR?

Reply:
Will do!